x-pack/plugins/canvas/public/state/middleware/es_persist.js

```diff
@@ -49,23 +50,30 @@ export const esPersistMiddleware = ({ getState }) => {
       return update(persistedWorkpad.id, persistedWorkpad).catch(err => {
         if (err.response.status === 400) {
           return notify.error(err.response, {
-            title: `Couldn't save your changes to Elasticsearch`,
+            title: i18n.translate('xpack.canvas.state.saveChanges.400ErrorMessageTitle', {
+              defaultMessage: "Couldn't save your changes to Elasticsearch",
+            }),
           });
         }
 
         if (err.response.status === 413) {
           return notify.error(
-            `The server gave a response that the workpad data was too large. This
-            usually means uploaded image assets that are too large for Kibana or
-            a proxy. Try removing some assets in the asset manager.`,
+            i18n.translate('xpack.canvas.state.workpadDataTooLarge.413ErrorMessage', {
+              defaultMessage:
+                'The server gave a response that the workpad data was too large. This usually means uploaded image assets that are too large for Kibana or a proxy. Try removing some assets in the asset manager.',
+            }),
             {
-              title: `Couldn't save your changes to Elasticsearch`,
+              title: i18n.translate('xpack.canvas.state.workpadDataTooLarge.413ErrorMessage', {
+                defaultMessage: "Couldn't save your changes to Elasticsearch",
+              }),
             }
           );
         }
 
         return notify.error(err.response, {
-          title: `Couldn't update workpad`,
+          title: i18n.translate('xpack.canvas.state.saveChanges.commonErrorMessageTitle', {
```

Comment:
xpack.canvas.state.esPersist.saveChangesToElasticsearch.400ErrorMessage
please note that we can use the same id for in different places if it is the same message